### DISCUSSION TITLE
Github's IP addresses for Pages have all been blacklisted

### DISCUSSION THREAD

**Discussion by Parking-Master** [community/community#106991] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Detected by [Spamhaus](https://check.spamhaus.org/listed/?searchterm=185.199.108.153), all of the following IPv4 addresses for GitHub Pages have been blacklisted:

- 185.199.108.153
- 185.199.109.153
- 185.199.110.153
- 185.199.111.153

As of such, I am unable to register new domains on my domain provider (FreeDNS.org).

*For the GitHub support team* - Please contact Spamhaus and request them to remove GitHub Pages' IP addresses from their listings. Thank you very much :)

**Update:** It looks like some users hosting their websites on GitHub Pages, are distributing Malware in their websites:

<hr>
<img width="787" alt="Screen Shot 2024-02-11 at 10 19 49 AM" src="https://github.com/community/community/assets/88283567/ba82202a-d5fe-4bc1-855e-d55959f275a3">
<hr>

GitHub must take down these websites/pages, and then contact Spamhaus and request them to remove the IP addresses from the blocklist:
> To have record SBL637883 (185.199.111.153/32) removed from the SBL, the Abuse/Security representative of fastly.com (or the Internet Service Provider responsible for supplying connectivity to 185.199.111.153/32) needs to [contact the SBL Team](mailto:sbl-removals@spamhaus.org?Subject=SBL637883%20185.199.111.153/32) by email [(use this link)](mailto:sbl-removals@spamhaus.org?Subject=SBL637883%20185.199.111.153/32) to explain how the abuse problem has been terminated (we need to know exactly how the issue has been dealt with and that this abuse problem is fully terminated). If the abuse problem that caused this listing has been terminated we will normally remove the listing from the SBL without delay.

**Comment by 0xdea** [comment]
I'm experiencing the same issue.

**Comment by Parking-Master** [comment]
@0xdea glad to hear it's not just me. I've contacted GitHub support and let them know with a little more information as well. They haven't responded yet.

**Reply by elswork** [reply]
I'm experiencing the same issue.

**Reply by Parking-Master** [reply]
GitHub still hasn't responded, and it's been 2 days! I don't know what they're doing. It's getting annoying that all of my free domains are down.

**Comment by Parking-Master** [comment, marked as answer]
Update: GitHub support has responded, they let me know that the suspicious websites hosted on Pages are no longer online, and that they've contacted Spamhaus to remove the IP addresses from the blocklist. It shouldn't take too long for the listings to be removed.

**Reply by katripathi-fadv** [reply]
any anticipation how much longer will it take?

**Reply by Parking-Master** [reply]
@Karunika they supposedly requested to take them off of the blacklist, but I don't know what's going on. Spanhaus may be slow or they won't accept the request for some reason.

**Reply by Parking-Master** [reply]
Update: Looks like they've all been removed from the blacklist. You can check here:

https://check.spamhaus.org/listed/?searchterm=185.199.108.153

Additionally I had to wait a few minutes for all of my projects to come back up one at a time. Thanks to GitHub support for handling this issue so easily.

**Comment by pubkey** [comment]
It happened again: https://github.com/orgs/community/discussions/143506

**Reply by LiteBrite82** [reply]
The pages team is on it; thanks @pubkey !!
